std/xml.d

```diff
@@ -2878,16 +2878,15 @@ class CheckException : XMLException
         this.err = err;
     }
 
-    private void complete(string entire) pure
+    private void complete(string entire) @safe pure
     {
-        import std.encoding : transcode;
         import std.string : count, lastIndexOf;
+        import std.utf : toUTF32;
```

Comment:
No comment here, just have to say YAY about getting rid of references to `std.encoding`.